requirements.txt

```diff
@@ -2,6 +2,7 @@ appdirs~=1.4
 coala_utils~=0.6.2
 colorlog~=2.7
 dependency_management~=0.3.1
+# ipdb
```

Comment:
@NiklasMM IIRC the `ipdb` requirement caused problems in CI. So I wanted to make it a dynamic `PipRequirement` when needed. That's why I also `import ipdb` only on demand. I will reproduce the actual problem again and paste it here...